solidity/random-beacon/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@keep-network/random-beacon-v2",
+  "version": "0.0.1-dev",
+  "description": "random beacon v2",
+  "scripts": {
+    "build": "hardhat compile",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "dependencies": {
+    "@openzeppelin/contracts": "^4.3.2",
+    "@tenderly/hardhat-tenderly": "^1.0.12",
+    "@thesis/solidity-contracts": "github:thesis/solidity-contracts#4985bcf"
```

Comment:
If we decide to remove `TestToken.sol` this entry could also be removed.

Reply:
I left it and added a sample test plus deployment script

Reply:
I'm not sure we need to add an external dependency if we want just to have a stub contract. Let's remove it.

Reply:
Ok, I think we will use it very soon anyways, but we can remove it for now.